src/rippled/accountState.js

```diff
@@ -110,13 +111,30 @@ const getAccountState = (account, rippledSocket) => {
         escrows: data[1],
         paychannels: data[2],
         xAddress: decomposedAddress || undefined,
```

Comment:
xAddress could still be valid in this account delete scenario.

Reply:
so we need to make another call to get the xAddress?

Reply:
https://livenet.xrpl.org/accounts/XVVFXHFdehYhofb7XRWeJYV6kjTEwboaHpB9S1ruYMsuXcG